bigtable-hbase-1.x-parent/bigtable-hbase-1.x-hadoop/pom.xml

```diff
@@ -76,6 +77,23 @@ limitations under the License.
       <groupId>io.dropwizard.metrics</groupId>
       <artifactId>metrics-core</artifactId>
       <version>${dropwizard.metrics.version}</version>
+      <exclusions>
+        <!-- see slf4j notes below -->
+        <exclusion>
+          <groupId>org.slf4j</groupId>
+          <artifactId>slf4j-api</artifactId>
+        </exclusion>
+      </exclusions>
+    </dependency>
+
+    <!-- unlike hbase-shaded-client, hbase-client depends on slf4j 1.6.1.
+      Which is older than the version requested by metrics-core. However,
+      metrics-core will work fine with the older version and we want to stay
+      compatible with hbase-client deps -->
+    <dependency>
+      <groupId>org.slf4j</groupId>
+      <artifactId>slf4j-api</artifactId>
+      <version>1.6.1</version>
```

Comment:
please double check this is in renovate config to skip

Reply:
I updated the renovate config